dev/config/bitcoind/bitcoin.conf

```diff
@@ -0,0 +1,19 @@
+rpcuser=rpcuser
```

Comment:
Is the intention to eventually remove these files from `core/api/dev`?

Reply:
yes